src/main/java/com/alipay/oceanbase/rpc/mutation/result/MutationResult.java

```diff
@@ -65,15 +82,28 @@ public long getAffectedRows() {
      */
     public Row getOperationRow() {
         Map<String, Object> rowsMap;
-        switch (result.getPcode()) {
```

Comment:
could you based on pcode to return affected rows directly?

Reply:
Fixed